crates/sui-replay/src/replay.rs

```diff
@@ -2129,6 +2129,7 @@ async fn prep_network(
         protocol_config.version.as_u64(),
     )
     .await;
+    *authority_state.execution_lock_for_reconfiguration().await = executed_epoch;
```

Comment:
Why was this not needed before but needed now?

Reply:
Because of the execution lock for signing stuff, which I've since removed. It is probably not necessary to pass the tests any more, but it seems clearly better to have the authority in the correct epoch during replay.